### ISSUE TITLE
Duende Version 7.0.5 breaks build

### ISSUE THREAD

**Issue by frankbolero** [dotnet/eShop#479]
Duende Version 7.0.5 breaks the build because of a vulnerability (https://github.com/advisories/GHSA-ff4q-64jc-gx98). A fix has been made in version 7.0.6 and this needs to be applied to this project in order for it to build correctly.

**Comment by adityamandaleeka** [comment]
This will be sorted out when [this PR](https://github.com/dotnet/eShop/blob/main/Directory.Packages.props#L11) is merged on August 13. For more information, check the discussion in that PR.

Until then, if you're blocked because of this error, you can update the version yourself in Directory.Packages.props: https://github.com/dotnet/eShop/blob/main/Directory.Packages.props#L11. If you're in an environment that then runs into the NU3042 error, you can update your environment to trust the Sectigo certificate.

**Comment by baskint** [comment]
yup, the suggested path fixes that, then i have another issue here. sorry for the duplicate.
https://github.com/dotnet/eShop/issues/492
